### PR TITLE
feat: Create config file from program

### PR DESCRIPTION
**Pull request by s-Sizz**
The user will be prompted to create a config file at `configPath` if:
* there is no file 
* username or password are default or empty values 
* using `--create-config` when running

### CODE REVIEW

File: CreateConfigFile.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import re
+from getpass import getpass
+from os.path import isfile
+from typing import Any
+
+import yaml
+from rich import print
+
+
```
Comment (DjordyKoert) [inline CreateConfigFile.py:9]: Remove these lines

File: CreateConfigFile.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
+import re
+from getpass import getpass
+from os.path import isfile
+from typing import Any
+
```
Comment (DjordyKoert) [inline CreateConfigFile.py:5]: Remove line

File: README.md [hunk 3]
```diff
@@ -2,11 +2,12 @@
 
 Are you tired of watching professional League of Legends games? Do you watch only for the drops? This is a revolution in the farming of League of Legends Esports capsules!
 
-This is a successor to the old [EsportsCapsuleFarmer](https://github.com/LeagueOfPoro/EsportsCapsuleFarmer) which relied on a web browser to watch videos. *Capsule Farmer Evolved* simulates traffic to lolesports.com servers and tricks it into thinking the account is watching a stream. This approach drastically lowers the hardware requirements.
+This is a successor to the old [EsportsCapsuleFarmer](https://github.com/LeagueOfPoro/EsportsCapsuleFarmer) which relied on a web browser to watch videos. _Capsule Farmer Evolved_ simulates traffic to lolesports.com servers and tricks it into thinking the account is watching a stream. This approach drastically lowers the hardware requirements.
```
Comment (DjordyKoert) [inline README.md:5]: What is the reason for changing this?

File: README.md [hunk 4]
```diff
@@ -2,11 +2,12 @@
...
 
 [Learn more about the esports drops directly from Riot games.](https://lolesports.com/article/lol-esports-2022-season-rewards-and-drops-update!/blt4ae38b4643f45741)
 
 ### Features
+
```
Comment (DjordyKoert) [inline README.md:10]: Also I am not sure why u added these lines after every heading or before code blocks? 

In my opinion they help keep the file readable

**Review by DjordyKoert — COMMENTED** [review]
Fix readme

File: main.py [hunk 5]
```diff
@@ -1,26 +1,38 @@
...
+    parser.add_argument(
+        "-c",
+        "--config",
+        dest="configPath",
+        default="./config.yaml",
+        help="Path to a custom config file",
+    )
+
+    parser.add_argument(
+        "--create-config", action="store_true", help="Create a config file"
```
Comment (DjordyKoert) [inline main.py:34]: Also place these parameters on new lines. Just like you did above

File: main.py [hunk 6]
```diff
@@ -1,26 +1,38 @@
+import argparse
+import sys
+from pathlib import Path
+from threading import Lock
+from time import sleep
+
```
Comment (DjordyKoert) [inline main.py:6]: Remove line
```suggestion
```

File: README.md (hunk 3 above)
Comment (s-Sizz) [reply README.md:5]: It was done by  Prettier. I only made manual changes in the config section

File: main.py [hunk 7]
```diff
@@ -1,26 +1,38 @@
...
-from pathlib import Path
-from time import sleep
-
 from Stats import Stats
 from VersionManager import VersionManager
 
-
 CURRENT_VERSION = 1.2
 
+
```
Comment (DjordyKoert) [inline main.py:20]: Remove line
```suggestion
```

File: README.md (hunk 3 above)
Comment (DjordyKoert) [reply README.md:5]: I think this part should be reverted then? I see no reason why this should be changed

**Comment by DjordyKoert** [comment]
also @s-Sizz just as a small disclaimer, I recommend naming your commits better `Update main.py` does not explain which changes you made.

**Pull request closed without merging**